egs/callhome_diarization/v1/local/make_callhome_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
You should put your name on the scripts you create, or do significant modification to. E.g.:

```
# Copyright 2016    <First name> <Last name>
# Apache 2.0.
```